.github/workflows/test_weaver-node-pkgs.yaml

```diff
@@ -131,24 +130,3 @@ jobs:
     - name: Tests
       run: npm run test
       working-directory: weaver/core/identity-management/iin-agent
-
-  build_docs:
```

Comment:
Do you want to add the build test of the cacti/docs? So that if some dependency gets upgraded by dependabot we may know if its not breaking the docs build.

Reply:
We have a build and publish action for the Cacti docs here: https://github.com/hyperledger/cacti/blob/main/.github/workflows/deploy_docs.yml. Isn't that sufficient?

Reply:
But that doesn't run on PRs, it is only used to publish from `main` branch after the PR is merged.

Reply:
OK, I take your point. I'll add a build action.